.env.example

```
@@ -37,7 +37,14 @@ FEATURE_GRAPHQL_COMMITS=false
 # FEATURE_GRAPHQL_PAGE_SIZE=100
 
 # Error Tracking Configuration
-# Optional: Sentry DSN for error tracking and monitoring
-# When configured, errors will be sent to Sentry for analysis
-# SENTRY_DSN=https://your-sentry-dsn@sentry.io/project-id
-# NEXT_PUBLIC_SENTRY_DSN=https://your-sentry-dsn@sentry.io/project-id
\ No newline at end of file
+# Sentry error tracking and performance monitoring
+# Organization: misty-step, Project: gitpulse
+SENTRY_DSN=https://724cc7295e04d879cd1aa8376faaf7b6@o4510313803677696.ingest.us.sentry.io/4510403670638592
+NEXT_PUBLIC_SENTRY_DSN=https://724cc7295e04d879cd1aa8376faaf7b6@o4510313803677696.ingest.us.sentry.io/4510403670638592
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SENTRY_DSN key should go before the SENTRY_DSN key

(UnorderedKey)